docs/source/testing/tox.rst

```diff
@@ -3,20 +3,90 @@
 ``tox``
 =======
 
+``tox`` is an automation project we use to run our testing scenarios. It gives us
+the ability to create a dynamic matrix of many testing scenarios, isolated testing environments
+and a provides a single entry point to run all tests in an automated and repeatable fashion.
+
+Documentation for tox can be found `here<https://tox.readthedocs.io/en/latest/>`_.
+
 
 .. _tox_environment_variables:
 
 Environment variables
 ---------------------
 
+When running ``tox`` we've allowed for the usage of environment variables to tweak certain settings
+of the playbook run using ``--extra-vars``. This is mostly useful for usage in Jenkins jobs or for using
+ceph-ansible to run manual tests. For example, if you wanted to test the latest master build of ceph using
+our ``xenial_cluster`` scenario you could do the following::
+
+    CEPH_DEV=true CEPH_DEV_BRANCH=master CEPH_DEV_SHA1=latest tox -rve jewel-ansible2.2-xenial_cluster
+
+The following environent variables are available for use:
+
+* ``FETCH_DIRECTORY`` : (default: ``changedir``) This would configure the ceph-ansible variable ``fetch_directory``. This defaults to
+  the ``changedir`` of the given scenario and should not need to be changed.
+
+* ``CEPH_RHCS`` : (default: ``false``) Setting this to ``true`` would enable testing of RHCS. This is set to ``true`` when using a ``rhcs-*``
+  testing scenario.
+
+* ``CEPH_ORIGIN``: (default: ``upstream``) This would configure the ceph-ansible variable ``ceph_origin``.
+
+* ``CEPH_DEV``: (default: ``false``) This would configure the ceph-ansible variable ``ceph_dev``.
+
+* ``CEPH_DEV_BRANCH``: (default: ``master``) This would configure the ceph-ansible variable ``ceph_dev_branch``.
```

Comment:
I think this would apply the same here. Since they are both named the same, it might be better to explain what the ceph-ansible variable does. For example:

    This would configure the ceph-ansible variable for the development branch in ceph.git, for example `wip-feature`